.github/workflows/crossdev.yml

```diff
@@ -17,6 +17,36 @@ jobs:
         target:
           - aarch64-unknown-linux-gnu
           - aarch64-unknown-linux-musl
+          - arm-unknown-linux-gnueabi
+          - arm-unknown-linux-gnueabihf
+          - arm-unknown-linux-musleabi
+          - arm-unknown-linux-musleabihf
+          - armv7-unknown-linux-gnueabi
+          - armv7-unknown-linux-gnueabihf
+          - mips-unknown-linux-gnu
+          - mips-unknown-linux-musl
+          - mipsel-unknown-linux-gnu
+          - mipsel-unknown-linux-musl
+          - mips64-unknown-linux-gnu
+          - mips64-unknown-linux-musl
+          - mips64el-unknown-linux-gnu
+          - mips64el-unknown-linux-musl
+          - powerpc-unknown-linux-gnu
+          - powerpc-unknown-linux-musl
+          - powerpc64-unknown-linux-gnu
+          - powerpc64-unknown-linux-musl
+          - riscv32-unknown-linux-gnu
+          - riscv32-unknown-linux-musl
+          - riscv64-unknown-linux-gnu
+          - riscv64-unknown-linux-musl
+          - sparc-unknown-linux-gnu
+          - sparc-unknown-linux-musl
```

Comment:
There is no sparc* port for musl (or musl port for sparc*), unfortunately.

Reply:
Good point, removed